release-notes/9.0/preview/api-diff/preview1/Microsoft.NETCore.App/9.0-preview1_System.Timers.md

```diff
@@ -0,0 +1,11 @@
+# System.Timers
+
+``` diff
+ namespace System.Timers {
+-    public class ElapsedEventArgs : EventArgs {
++    public sealed class ElapsedEventArgs : EventArgs {
```

Comment:
That's a breaking change, isn't it?

Reply:
No, it wasn't publicly constructible before, so no one could have derived from it.

Reply:
Oh didn't notice. Thanks!